pkg/controller/podautoscaler/replica_calculator.go

```diff
@@ -29,29 +30,32 @@ import (
 	v1coreclient "k8s.io/client-go/kubernetes/typed/core/v1"
 	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
 	metricsclient "k8s.io/kubernetes/pkg/controller/podautoscaler/metrics"
+	"runtime/debug"
 )
 
 const (
 	// defaultTestingTolerance is default value for calculating when to
 	// scale up/scale down.
-	defaultTestingTolerance = 0.1
-
-	// Pod begins existence as unready. If pod is unready and timestamp of last pod readiness change is
-	// less than maxDelayOfInitialReadinessStatus after pod start we assume it has never been ready.
-	maxDelayOfInitialReadinessStatus = 10 * time.Second
+	defaultTestingTolerance                     = 0.1
+	defaultTestingCpuTaintAfterStart            = 2 * time.Minute
```

Comment:
a) you need a comment as to why this is two minutes
b) if this is based on the metrics-server collection window, it shouldn't be constant -- it should be using the reported window size from metrics-server.  If this is "just a guess", that's not a good answer.

Reply:
thought you left this comment in a different commit and were referring to
values HPA uses (which were hard coded in the previous commit).

Those are just values I use for testing, it doesn't matter much what exactly they are.